Core/include/Acts/TrackFitting/GaussianSumFitter.hpp

```diff
@@ -230,9 +230,9 @@ struct GaussianSumFitter {
             .status();
 
     if (intersectionStatusStartSurface != Intersection3D::Status::onSurface) {
-      ACTS_ERROR(
-          "Surface intersection of start parameters with bound-check failed");
-      return GsfError::StartParametersNotOnStartSurface;
+      ACTS_DEBUG(
+          "Surface intersection of start parameters WITH bound-check failed");
+      // return GsfError::StartParametersNotOnStartSurface;
```

Comment:
If you think that's production setup like this, you should remove the comment.

Reply:
Yeah that comment should be removed.